src/main/java/net/glowstone/entity/GlowEntity.java

```diff
@@ -318,10 +320,20 @@ public GlowEntity(Location location) {
         server = world.getServer();
```

Comment:
Could you do the suggested change of moving these two lines (assigning `world` and `server`) to after the `if` statement? That would fix a bug where the event sets the location to another world.

Reply:
Okay